api/instance/v1/instance_sdk.go

```diff
@@ -700,6 +700,18 @@ type Dashboard struct {
 	IPsUnused uint32 `json:"ips_unused"`
 }
 
+type ExtraNetworks struct {
+	Address string `json:"address,omitempty"`
```

Comment:
Why is there omit empty in a response message ?

Reply:
Because `ExtraNetworks` is used in request and response messages. I will create an `ExtraNetworksTemplate`.